discovery/device/src/OnboardingInfo.cs

```diff
@@ -18,14 +18,14 @@ public class OnboardingInfo
         /// <summary>
         /// Credentials to authenticate with the provisioning service
         /// </summary>
-        public X509Certificate2 ProvisioningCertificate { get; private set; }
+        public X509Certificate2Collection ProvisioningCertificate { get; private set; }
```

Comment:
Why this change?

Reply:
The API was changed from giving us just one certificate to provided the entire cert chain. The client facing code was changed to reflect this.